src/test/java/com/mycompany/carina/demo/mytests/SauceDemoTest.java

```diff
@@ -60,9 +60,8 @@ public void testCheckout() {
         checkoutYourInformationPage.typePostalCodeInput("12345");
         checkoutYourInformationPage.clickContinueButton();
         CheckoutOverviewPage checkoutOverviewPage = new CheckoutOverviewPage(getDriver());
-        checkoutOverviewPage.clickFinishButton();
-        CheckoutCompletePage checkoutCompletePage = new CheckoutCompletePage(getDriver());
-        Assert.assertTrue(checkoutCompletePage.isPageOpened(), "Checkout is not complete!");
+        // assert on this page
+        Assert.assertEquals(checkoutOverviewPage.getProductsCount(), 1, "Incorrect amount of products found.");
```

Comment:
check full checkout flow, remove comment
```suggestion
        // assert on this page
        Assert.assertEquals(checkoutOverviewPage.getProductsCount(), 1, "Incorrect amount of products found.");
        CheckoutCompletePage checkoutCompletePage = checkoutOverviewPage.clickFinishButton();
        Assert.assertTrue(checkoutCompletePage.isPageOpened(), "Checkout is not complete!");
```